File: src/Omnia.ProcessManagement.Models/Processes/ProcessDataWithAuditing.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace Omnia.ProcessManagement.Models.Processes
{
    public class ProcessDataWithAuditing : ProcessData
    {
        public string CreatedBy { get; set; }
        public string ModifiedBy { get; set; }
        public DateTimeOffset CreatedAt { get; set; }
        public DateTimeOffset ModifiedAt { get; set; }
    }
}

File: src/Omnia.ProcessManagement.Web/client/fx/models/data/processes/ProcessDataWithAuditing.ts
import { CanvasDefinition } from '../drawingdefinitions';
import { MultilingualString } from '@omnia/fx-models';
import { ProcessData } from '.';

export interface ProcessDataWithAuditing extends ProcessData {
    createdBy: string;
    modifiedBy: string;
    createAt: Date;
    modifiedAt: Date;
}